packages/ngtools/webpack/src/transformers/replace_resources.ts

```diff
@@ -159,9 +159,10 @@ function visitComponentMetadata(
       return undefined;
 
     case 'templateUrl':
+      const loaderOptions = moduleKind < ts.ModuleKind.ES2015 ? '?esModule=false' : '';
```

Comment:
Is `ts.ModuleKind` intended to be comparable like this? I'm wondering if there is a way this could break with future changes to `ts.ModuleKind`? I guess any changes will (probably) come > ES2015 and it is unlikely (though not impossible) that those could would be incompatible with ESM.

I'm wondering if it might be better to explicitly call out the specific modules where we want to use `?esModule=false`? I don't feel too strongly about this, I just noticed the `<` comparison on an enum which made me wonder if that could bite us later.

Reply:
The check is already present in https://github.com/angular/angular-cli/blob/5305e00cb8082dfca8364debf2091097e0f1f7e9/packages/ngtools/webpack/src/transformers/replace_resources.ts#L263.

If TypeScript changes the enum values, I think we can revisit this, considering we don’t support future TypeScript versions.

Reply:
SGTM, no need to make this more complicated than necessary right now.

Reply:
Ps: I don’t feel strongly about not changing it either way.